resource-definitions/terraform-driver/co-provision/README.md

```diff
@@ -0,0 +1,7 @@
+## Resource co-provisioning
+
+This section contains an example of Resource Definitions using the [Terraform Driver](https://developer.humanitec.com/integration-and-extensions/drivers/generic-drivers/terraform/) and illustrating co-provisioning concept.
+
+Scenario: For each AWS S3 bucket resource an AWS IAM policy resource must be created. The bucket properties (region, ARN) should be passed to the policy resource. 
+In other words, an IAM Policy resource depends on a S3 resource, but it needs to be created implicitly.
```

Comment:
```suggestion
In other words, an IAM Policy resource depends on a S3 resource, but it needs to be created automatically.
```

Reply:
I suggest changing the wording from "implicitly" to "automatically". While "implicit" describes the mechanism well, that term is defined differently in the [Resource Type context](https://developer.humanitec.com/platform-orchestrator/reference/resource-types/#resource-type-use). Changing it might prevent confusion.